pkg/trace/api/info_test.go

```diff
@@ -246,9 +246,9 @@ func TestInfoHandler(t *testing.T) {
 			h.ServeHTTP(rec, req)
 			assert.Equal(t, rec.Body.String(), tt.expected)
 			if rec.Body.String() != tt.expected {
-				t.Fatal("Output of /info has changed. Changing the keys "+
+				t.Fatalf("Output of /info has changed. Changing the keys "+
 					"is not allowed because the client rely on them and "+
-					"is considered a breaking change:\n\n%f", rec.Body.String())
+					"is considered a breaking change:\n\n%s", rec.Body.String())
```

Comment:
This is an unrelated fix. I can revert if desired.

Reply:
This is good.